color-image-database/colorconverter.h

```diff
@@ -0,0 +1,14 @@
+#ifndef COLORCONVERTER_H
+#define COLORCONVERTER_H
+
+
+#include "color.h"
+#include "qcolor.h"
+
+class ColorConverter
+{
+public:
+    static QColor hexToColor(Color color);
```

Comment:
I think it would make more sense to simply have this method be a part of the Color class, smth like "toColor()"